CMSgov/onemac
# remove override of actionLabel for amendments

Story: https://qmacbis.atlassian.net/browse/OY2-22594
Endpoint: See github-actions bot comment

### Details

Remove override of title for amendments. Keep default "Package Actions"

### Changes

- Removed "actionLabel" property from waiverAmendment config
- Removed redundant "actionLabel" property from waiverAppendixKDetail as it was same as in defaultDetail


### Test Plan

1. Login as any user with package view access
2. Navigate to any waiver package details page
3. Verify Label change
